### DISCUSSION TITLE
App-of-Apps pattern not working for other namespaces

### DISCUSSION THREAD

**Discussion by Rajendra08** [argoproj/argo-cd#8684] [General]
Hi Everyone,

We have 4 helm charts and we would like to deploy these 4 helm charts in two different namespaces.
We thought app-of-apps pattern suits to our need.

![app-of-apps](https://user-images.githubusercontent.com/47872402/156796582-b5cd4fbb-3525-43a3-a7fd-146352f369a3.png)

We have created in argoCD. argoCD shows it is created and synced.
But I dont see any resources in namespace1 and namespace2.

If I deploy same app-of-apps in argocd namespace (all 4 charts), then it deploys and working fine.

Please let us know your inputs on the same.

Thanks

Regards,
Rajendra A

**Comment by todaywasawesome** [comment]
See this previous issue. https://github.com/argoproj/argo-cd/issues/696 Support was added a while ago.
